File: Tugas coding modul 4/coding c/PRAK403-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void main(){
    int x, y, i, j;

    printf("Input \n");
    scanf("%d %d", &x, &y);

    printf("\nOutput \n");
    if (x < y){
        for(i = x; i <= y; i++){
            j = y; 
            j -= (i - x);
            printf("%d %d", i, j);
            if (i == y){
                break;
            } else{
                printf(" - ");
            }
        } 
    }

    if (x > y){
        for(i = x; i >= y; i--){
            j = y; 
            j += (x - i);
            printf("%d %d", i, j);
            if (i == y){
                break;
            } else{
                printf(" - ");
            }
        } 
    }
}

File: Tugas coding modul 4/coding py/PRAK403-2310817120005-Noviana_Nur_Aisyah.py
x, y = map(int, input("Input \n").split())

print("\nOutput")
if x < y:
    for i in range(x, y + 1, +1): # tidak pakai +1 (increment) juga tidak apa
        j = y - (i - x)
        print(i, j, end=" - " if i != y else "")
else:
    for i in range(x, y - 1, -1):
        j = y + (x - i)
        print(i, j, end=" - " if i !=y else "")